extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -114,6 +122,17 @@ public void updateClasspath(IProject project, IProgressMonitor monitor) throws C
                 new File(jvmBuildTarget.getJavaHome()));
         classpath.add(JavaCore.newContainerEntry(JavaRuntime.newJREContainerPath(vm)));
 
+
+        // Use a map to deduplicate dependencies which point to the same jar.
+        Map<String, IClasspathEntry> dependencyMap = new HashMap<>();
```

Comment:
The order of classpath entries is random. It's better to keep the order same as used in the build target.

Reply:
Using LinkedHashMap instead.